src/core/attributeformmodelbase.cpp

```diff
@@ -102,14 +113,12 @@ void AttributeFormModelBase::setFeatureModel( FeatureModel *featureModel )
   if ( mFeatureModel )
   {
     disconnect( mFeatureModel, &FeatureModel::currentLayerChanged, this, &AttributeFormModelBase::onLayerChanged );
-    disconnect( mFeatureModel, &FeatureModel::featureChanged, this, &AttributeFormModelBase::onFeatureChanged );
```

Comment:
Without going into it deep I wonder - when the connection is not needed - if the signal is needed at all. Means, is it needed that we connect the `modelReset` here https://github.com/opengisch/QField/pull/1073/files#diff-20da0d6569bb292b15438db05c686d8fR31 and is it needed to emit this signal after the model is reseted here https://github.com/opengisch/QField/pull/1073/files#diff-20da0d6569bb292b15438db05c686d8fR59 ?

Reply:
The onFeatureChanged signal is needed for the QML property.

Reply:
Ah okay. Do we need to connect it in the FeatureModel constructor?

Reply:
Na, no need for it there.